media-libs/zita-convolver/zita-convolver-2.0.0-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils flag-o-matic multilib toolchain-funcs
+
+DESCRIPTION="C++ library implementing a real-time convolution matrix"
+HOMEPAGE="http://kokkinizita.linuxaudio.org/linuxaudio/downloads/index.html"
+SRC_URI="http://kokkinizita.linuxaudio.org/linuxaudio/downloads/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm"
+IUSE="cpu_flags_x86_mmx cpu_flags_x86_sse cpu_flags_x86_sse2"
+
+RDEPEND="sci-libs/fftw:3.0"
+DEPEND="${RDEPEND}"
+
+S=${WORKDIR}/${P}/libs
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-makefile.patch
+}
+
+src_compile() {
+	# upstream defaults to this on x86 but patched out of the Makefile
+	# try to reenable optimisation for x86 and allow building on other arch's
+	use cpu_flags_x86_mmx && append-flags "-mmmx"
+	use cpu_flags_x86_sse && append-flags "-msse" "-mfpmath=sse"
+	use cpu_flags_x86_sse2 && append-flags "-msse2"
+	append-flags "-ffast-math"
```

Comment:
`-ffast-math` is fine to set if program really doesn't depend on any of the fancy features. However, I'd say upstream should set it via configure/makefile rather than us.